cli/Gopkg.lock

```diff
@@ -0,0 +1,56 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
+
+
+[[projects]]
+  name = "github.com/bitly/go-simplejson"
+  packages = ["."]
+  revision = "aabad6e819789e569bd6aabf444c935aa9ba1e44"
+  version = "v0.5.0"
+
+[[projects]]
+  name = "github.com/fatih/color"
+  packages = ["."]
+  revision = "9aae6aaa22315390f03959adca2c4d395b02fcef"
+
+[[projects]]
+  name = "github.com/hashicorp/go-version"
+  packages = ["."]
+  revision = "03c5bf6be031b6dd45afec16b1cf94fc8938bc77"
+
+[[projects]]
+  name = "github.com/kardianos/osext"
+  packages = ["."]
+  revision = "29ae4ffbc9a6fe9fb2bc5029050ce6996ea1d3bc"
+
+[[projects]]
+  name = "github.com/mattn/go-colorable"
+  packages = ["."]
+  revision = "4af63d73f5bea08b682ad2cd198b1e607afd6be0"
+
+[[projects]]
+  name = "github.com/mattn/go-isatty"
+  packages = ["."]
+  revision = "56b76bdf51f7708750eac80fa38b952bb9f32639"
+
+[[projects]]
+  name = "github.com/urfave/cli"
+  packages = ["."]
+  revision = "1efa31f08b9333f1bd4882d61f9d668a70cd902e"
+  version = "v1.18.0"
+
+[[projects]]
+  name = "golang.org/x/sys"
+  packages = ["unix"]
+  revision = "eb2c74142fd19a79b3f237334c7384d5167b1b46"
+
+[[projects]]
+  name = "gopkg.in/yaml.v2"
+  packages = ["."]
+  revision = "cd8b52f8269e0feb286dfeef29f8fe4d5b397e0b"
+
+[solve-meta]
```

Comment:
Not sure where it belongs, but depending readme recommends ci should use a fixed version.

Reply:
`dep` readme, not depending